npm/azure-npm.yaml

```diff
@@ -114,3 +114,17 @@ spec:
           path: /etc/protocols
           type: File
       serviceAccountName: azure-npm
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: npm-metrics-cluster-service
+  namespace: kube-system
+  labels:
+    app: npm-metrics
+spec:
+  selector:
+    k8s-app: azure-npm
+  ports:
+    - port: 9000
```

Comment:
What does this port 9000 do?

Reply:
this is just the port for the service (unrelated to the node/host). So you can access this service (which relays a random NPM pod) on port 9000 i.e. `curl npm-metrics-cluster-service.kube-system:9000/metrics`